File: finah-desktop/Klassen/finah_desktop/Account.java
package finah_desktop;

public class Account extends SuperklasseAanvraagAccount {

	private EID gekoppelde_eid;
	public EID getGekoppelde_eid() {
		return gekoppelde_eid;
	}

	public void setGekoppelde_eid(EID gekoppelde_eid) {
		this.gekoppelde_eid = gekoppelde_eid;
	}

	public Account() {
	}

	public Account(String naam, String voornaam, String rijksregisterNr,
			String adres, int telefoon, int gsm, String login, String password,
			String email, String geheimeVraag, String geheimAntwoord,TypeAccount typeAcc) {
		super(naam, voornaam, rijksregisterNr, adres, telefoon, gsm, login,
				password, email, geheimeVraag, geheimAntwoord,typeAcc);

	}

	/* (non-Javadoc)
	 * @see java.lang.Object#hashCode()
	 */
	@Override
	public int hashCode() {
		final int prime = 31;
		int result = super.hashCode();
		result = prime * result
				+ ((gekoppelde_eid == null) ? 0 : gekoppelde_eid.hashCode());
		return result;
	}

	/* (non-Javadoc)
	 * @see java.lang.Object#equals(java.lang.Object)
	 */
	@Override
	public boolean equals(Object obj) {
		if (this == obj)
			return true;
		if (!super.equals(obj))
			return false;
		if (getClass() != obj.getClass())
			return false;
		Account other = (Account) obj;
		if (gekoppelde_eid == null) {
			if (other.gekoppelde_eid != null)
				return false;
		} else if (!gekoppelde_eid.equals(other.gekoppelde_eid))
			return false;
		return true;
	}
	
}

File: finah-web/PHP/Models/Account.php
<?php
    require_once "SuperKlasseAanvraagAccount.php";

    /**
     * Created by PhpStorm.
     * User: Gert
     * Date: 24/03/2015
     * Time: 15:57
     */
    class Account extends SuperKlasseAanvraagAccount
    {

        public function __construct($Id = null, $Naam = null, $Voornaam = null, $Adres = null, $Telnr = null, $Login = null, $Passwd = null, $Email = null, $TypeAcc = null)
        {
            parent::__construct($Id, $Naam, $Voornaam,$Adres, $Telnr, $Login, $Passwd, $Email, $TypeAcc);
        }
    }